R/outcome_bin_logistic.R

```diff
@@ -89,3 +89,13 @@ logistic_bin_outcome <- function(binary_var,
     )
   )
 }
+
+
+#' Legacy function for binary logistic regression
+#'
+#' Please use `outcome_bin_logistic()` instead.
+#' @param ... Deprecated arguments to `logistic_bin_outcome`.
+#' @export
+logistic_bin_outcome <- function(...) {
+  stop("`logistic_bin_outcome()` is deprecated. Use `outcome_bin_logistic()` instead.")
```

Comment:
There are these special functions we could use here: https://stat.ethz.ch/R-manual/R-devel/library/base/html/Defunct.html

Dunno if it adds anything really...